rasa_utils/bot_server_channel.py

```diff
@@ -85,10 +87,8 @@ def send_image_url(self, recipient_id, image_url):
 
 class BotServerInputChannel(InputChannel):
 
-    app = Klein()
-
     def __init__(
-        self, agent, port=5002, static_files=None, message_store=FileMessageStore()
+        self, agent, preprocessor=None, port=5002, static_files=None, message_store=FileMessageStore()
```

Comment:
We should provide a default argument for the `agent`. Otherwise the channel can not be initialised using the new `credentials.yml` method (https://rasa.com/docs/core/connectors/#managing-credentials). `agent=None` should suffice for now.